config/database.js

```diff
@@ -18,13 +18,14 @@ module.exports = {
 
   /*
   |--------------------------------------------------------------------------
-  | Sqlite
+  | Drivers
```

Comment:
because from the docs, some dbs have more than one driver, so its left for the user to decide which driver he wants to install/use